### PR TITLE
Added JetBrains YouTrack version 21.3.21558.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21845)

### CODE REVIEW

File: manifests/j/JetBrains/YouTrack/21.3.21558/JetBrains.YouTrack.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Copyright: Copyright (c) JetBrains s.r.o.
+ShortDescription: The project management tool designed for agile teams.
+Description: Enjoy having everything together in one place. Track tasks, manage projects, keep a knowledge base, collaborate with your team, and deliver great products. Designed with all the members of your team in mind. Free for teams of 10. In our cloud or on your server.
+Moniker: youtrack
+Tags:
+- issue tracker
+- time management
+- reports
+- knowledge base
+- development
```
Comment (vedantmgoyal2009) [inline manifests/j/JetBrains/YouTrack/21.3.21558/JetBrains.YouTrack.locale.en-US.yaml:22]:
```suggestion
- issue-tracker
- time-management
- reports
- knowledge-base
- development
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

File: manifests/j/JetBrains/YouTrack/21.3.21558/JetBrains.YouTrack.locale.en-US.yaml (hunk 1 above)
Comment (vedantmgoyal2009) [reply manifests/j/JetBrains/YouTrack/21.3.21558/JetBrains.YouTrack.locale.en-US.yaml:22]: https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#optional-field-10

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**